### PR TITLE
refractor(super): use barrel to rollup exports

### PR DESCRIPTION
**Pull request by pickfire**

### CODE REVIEW

**Comment by dwieeb** [comment]
@pickfire What is barrel?

**Comment by pickfire** [comment]
A term taken from angular to re-exports the module.

https://angular.io/guide/glossary#barrel

File: ionic-angular/official/super/src/app/app.module.ts [hunk 1]
```diff
@@ -10,9 +10,9 @@ import { TranslateHttpLoader } from '@ngx-translate/http-loader';
 import { IonicApp, IonicErrorHandler, IonicModule } from 'ionic-angular';
 
 import { Items } from '../mocks/providers/items';
-import { Settings } from '../providers/providers';
-import { User } from '../providers/providers';
-import { Api } from '../providers/providers';
+import { Settings } from '../providers';
+import { User } from '../providers';
+import { Api } from '../providers';
```
Comment (mraible) [inline ionic-angular/official/super/src/app/app.module.ts]: You could put these all on one line.

**Comment by dwieeb** [comment]
@pickfire Apologies for the activity. I've been trying to make it break in CI 😆 

Looks like there's some import errors (see the "test" step), could you clean those up? I'm also getting them locally.

**Comment by dwieeb** [comment]
Also, thanks for the link to the barrel docs. I'm not an Angular expert 😄

**Comment by pickfire** [comment]
@mraible Thanks for the suggestion. Updated. :smiley: 

@dwieeb Welcome, test fixed. I am not an angular expert either, I just happened to saw it in the docs. :smile:

**Comment by dwieeb** [comment]
Awesome, @pickfire !!

**Pull request merged**